### DISCUSSION TITLE
Thank you SO much!

### DISCUSSION THREAD

**Discussion by MrMAnderson** [mrencyc/ComicControl#2] [General]
I have been looking EVERYWHERE for an updated version of ComicControl because I've been wanting to make my own webcomic site for a long time. I tried doing some self-hosting to prototype things on my computer and when I used the files from the main website. Nothing really worked. When I tried your fork, it worked!!!  (though I do still see some small errors pop up on my page here and there, I presume it's something on my end, I'm self hosting using XAMPP, so maybe there's something I'm missing. If the errors are something else , at least I finally have a place to discuss this CMS)

Thank you so much for this fork! Gives people like me an opportunity to have a good art/comic website on our own terms!

**Comment by mrencyc** [comment]
I'm glad it's working out for you! I wouldn't be so sure the errors are coming from you, updating to the latest PHP introduced a lot of bugs that I still haven't fully ironed out, and I suspect there's still a lot of stuff I overlooked in the install process since most of my work is done on an already initialized version. Feel free to let me know about any errors that pop up and I can take a look to see if there's anything I can do.
